site/pages/api/create-consequence-network.api.ts

```diff
@@ -25,13 +24,12 @@ import { getSession } from "../../utils/apiUtils/auth";
 
 const createConsequenceNetwork = async (req: NextApiRequest, res: NextApiResponse): Promise<void> => {
     try {
-        const isFromTemplate = isDisruptionFromTemplate(req);
         const { template, addAnotherConsequence } = req.query;
         const body = req.body as NetworkConsequence;
         const validatedBody = networkConsequenceSchema.safeParse(body);
         const session = getSession(req);
 
-        const { draft } = req.query;
+        const { draft, isFromTemplate } = req.query;
```

Comment:
We should ensure isFromTemplate is passed through as a query param wherever the relevant api endpoints are called.

For example, in site/pages/type-of-consequence/[disruptionId]/[consequenceIndex].page.tsx on line 54 the form action should be:
```action={`/api/type-of-consequence${isFromTemplate ? "?isFromTemplate=true" : ""}`}```

Reply:
I believe it is in the api when it uses the redirectwithqueryparams

Reply:
Fixed as part of commit 4d8fa60